mono/utils/mono-error.h

```diff
@@ -56,7 +56,7 @@ __pragma(warning (disable:4201))
 #endif
 
 /*Keep in sync with MonoErrorInternal*/
-typedef union _MonoError {
+typedef union MonoErrorExternal {
```

Comment:
Do we care about C++ name mangling compatibility?
If so put this back to `_MonoError`.